src/transifex_api/__init__.py

```diff
@@ -139,3 +142,45 @@ class TeamMembership(jsonapi.Resource):
 @_api.register
 class ResourceLanguageStats(jsonapi.Resource):
     TYPE = "resource_language_stats"
+
+
+@_api.register
+class ResourceStringsAsyncDownload(jsonapi.Resource):
+    TYPE = "resource_strings_async_downloads"
```

Comment:
I think this endpoint does not exist in APIv3. At least I couldn't find it in the latest devel.

Reply:
You are right! And I don't even use it in Zendesk. Removed